README_GPDB_ru.md

```diff
@@ -0,0 +1,207 @@
+# 1. Поддерживаемые версии Greenplum
```

Comment:
I think some introduction is required at the beginning of the document in order to describe what is the intention of this 'howto'.